packages/client/vue.config.js

```diff
@@ -11,6 +11,16 @@ module.exports = {
       options.patterns[0].globOptions.ignore.push('**/deploy-config.js');
       return [options];
     });
```

Comment:
Adding the new router unit test revealed the following error when tests were run:

```
WEBPACK  Failed to compile with 1 error(s)

Error in ./src/components/SearchFilter.vue?vue&type=style&index=0&id=7c053370&lang=scss&

  Module build failed (from ../../node_modules/sass-loader/dist/cjs.js):
  SassError: Can't find stylesheet to import.
    ╷
  2 │ @import '../../scss/colors-semantic-tokens.scss';
    │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
    Users/nason/Jobs/USDR/Repositories/usdr-gost/packages/client/src/components/SearchFilter.vue 2:9  root stylesheet
```

I think this appeared now because the router necessarily imports a lot of components so the router test was the first test to (indirectly) reference `SearchFilter`.